modules/virt-deleting-vmis-web.adoc

```diff
@@ -0,0 +1,14 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virt-manage-vmis.adoc
+
+[id="virt-deleting-vmis-web_{context}"]
+= Deleting a standalone virtual machine instance using the web console
+
+
+.Procedure
+
+. In the {VirtProductName} console, click *Workloads* -> *Virtual Machines* from the side menu.
+. Click the &#8942; button of the standalone virtual machine instance (VMI) that you want to delete and select *Delete Virtual Machine Instance*.
+** Alternatively, click the standalone VMI name to open the *Virtual Machine Details* screen and click *Actions* -> *Delete Virtual Machine Instance*.
```

Comment:
Click the VMI name is opening the `Overview` screen, not `Details`.

Reply:
Please review the content changes.

Reply:
I did not see your response to my previous comment above, therefore I am asking. Was the updated content accurate? If it is not, please advise what is the correct product behavior. Thanks.

Reply:
Looks good.